packages/dom/src/lib/ElementAssertion.ts

```diff
@@ -90,4 +90,95 @@ export class ElementAssertion<T extends Element> extends Assertion<T> {
       invertedError,
     });
   }
+
+  /**
+   * Asserts that the element has the specified class.
+   *
+   * @param className - The class name to check.
```

Comment:
No need to add the hyphen. IDEs add it for us:
```suggestion
   * @param className The class name to check.
```